Source/WebCore/css/calc/CSSCalcValue.cpp

```diff
@@ -246,6 +246,8 @@ static RefPtr<CSSCalcExpressionNode> createCSS(const CalcExpressionNode& node, c
             auto children = createCSS(operationChildren, style);
             if (children.size() == 2)
                 return CSSCalcOperationNode::createStep(op, WTFMove(children));
+            else if (CalcOperator::Mod == op || CalcOperator::Rem == op)
```

Comment:
Rather than add tests involving `op` here, why not break them out of this switch block?

Reply:
Yeah, fair enough, I was just trying to reduce duplicate code, but I agree this style is a little odd. I'll change it now